docker-compose.production.yml

```diff
@@ -82,6 +82,20 @@ services:
     secrets:
       - petabox_seed
 
+  solr_updater:
+    # restart: always
+    hostname: "$HOSTNAME"
+    environment:
+      - OL_CONFIG=/olsystem/etc/openlibrary.yml
+      - SOLR_UPDATER_STATE_FILE=/1/var/run/openlibrary/solr-update.offset
+      - SOLR_UPDATER_EXCLUDE_EDITS_CONTAINING=ImportBot
```

Comment:
Some of these parameters have changed; let me push up some of the local mods from ol-home.